developers/staking.md

```diff
@@ -26,6 +27,6 @@ Currently, the following staking interfaces exist for the
 
 Just connect your wallet to get started!
 
+- [https://alphab.ai/s/t/celestia-mocha-4/](https://alphab.ai/s/t/celestia-mocha-4/)
 - [https://explorer.kjnodes.com/celestia-testnet](https://explorer.kjnodes.com/celestia-testnet)
-- [https://test.anode.team/celestia](https://test.anode.team/celestia)
```

Comment:
hi @Northa why is this removed?

Reply:
hi @jcstein have you tried to open that link? 
On my side it throws _Sorry, The chain is not found_  
Maintainer probably removed celestia testnet i.e link is no longer valid. 
So I removed it to keep a cleaner look

Reply:
thank you! good catch.